guides/v2.3/graphql/reference/paypal.md

```diff
@@ -0,0 +1,231 @@
+---
+group: graphql
+title: Paypal endpoint
+---
+
+The `Paypal` endpoint provides support for the following PayPal payment methods:
+
+* PayPal Express Checkout
+* PayPal Payflow Pro and PayPal Payflow Link
+
+The following steps describe the flow of calls required to complete a typical PayPal checkout transaction. A successful purchase requires that you send three mutations to PayPal, and the buyer must approve the purchase by logging in to PayPal.
+
+1. **Send a token request.** When the buyer clicks a PayPal button, the frontend executes the `createPaypalExpressToken` mutation. The Magento `PaypalGraphQl` module gathers information in the specified cart and sends this information to PayPal.
+
+2. **PayPal sends a token.** If the token request succeeds, PayPal returns a token and a payer ID. PayPal also sends payment-related data that is outside the scope of GraphQL. You must include this token in subsequent steps. The buyer is redirected to the payment confirmation page, which was specified in the token request.
+
+3. **Redirect the customer to PayPal for approval.** Depending on your implementation, the buyer is either redirected to the PayPal login screen, or the buyer enters their credentials in-context. 
+
+4. **PayPal redirects the customer back to your site.** If the customer approves the payment, PayPal directs the customer back to the payment confirmation page.
+
+5. **Set the payment method.** The frontend runs the `setPaymentMethodOnCart` mutation. The payload includes the PayPal token and the payer ID. The cart may have been updated since the token was requested with shipping costs, taxes, and other adjustments. Magento submits the updated cart to PayPal.
+
+6. **Complete the transaction.** Place the order with the `placeOrder` mutation. PayPal captures the payment by transferring the funds from the customer account to the appropriate merchant account. Magento creates an order, ready for fulfillment.
+
+## Mutation
+
+The `createPaypalExpressToken` mutation initiates a PayPal checkout transaction and receives a token. You must specify this token when setting a PayPal payment method.
+
+## Syntax
+
+`createPaypalExpressToken(input: PaypalExpressTokenInput): PaypalExpressToken`
```

Comment:
Other mutations did this syntax formatting:
mutation: {createCustomer(input: CustomerInput!) {CustomerOutput}}

has this convention changed or are the PayPal mutations different?

Reply:
It's different